tests/Constraints/FormatTest.php

```diff
@@ -23,7 +23,8 @@ public function testNullThing()
         $validator = new FormatConstraint();
         $schema = new \stdClass;
 
-        $validator->check('10', $schema);
+        $checkValue = 10;
+        $validator->check($checkValue, $schema);
```

Comment:
Yes, because the individual validators *do* require a reference. The references are needed for anything that modifies the input value, including type coercion and setting default values.

The line you've highlighted here tests `FormatConstraint` directly, not the top-level `Validator` class.